geotags/views.py

```diff
@@ -47,16 +49,18 @@
             ORDER BY sounds_sound.created DESC
                      %(end)s"""
 
-def generate_json(sound_queryset):
-    # When using Django Sound queries:
-    #sounds_data = [[s.id, s.geotag.lat, s.geotag.lon] for s in sound_queryset]
+def generate_bytearray(sound_queryset):
+    # sounds as bytearray
+    packed_sounds = cStringIO.StringIO()
+    for sound in sound_queryset:
+        packed_sounds.write(struct.pack("i", sound.id))
+        packed_sounds.write(struct.pack("i", int(sound.lat*1000)))
```

Comment:
https://en.wikipedia.org/wiki/Decimal_degrees lists approximations of scales. I wouldn't go higher than 5dp, maybe 4 is OK too.